build.sh

```diff
@@ -3,11 +3,11 @@ set -euo pipefail
 
 NCC_OPTS=""
 if [ -z "${DEV-}" ]; then
-  NCC_OPTS="-m -s"
```

Comment:
I thought this `-m -s` already enabled sourcemap with minification, although it doesn't work as expect. 🤔

Reply:
Right, but this if branch is for production, not dev builds.

Reply:
Wat? Oh I got it, I was wondering what's that trailing `-` in `if` 😂